### ISSUE TITLE
[1.9.24] iPhone 6 / 10.0.1 - Application Crash / Black frames display - Create lomotif with 100+ images

### ISSUE THREAD

**Issue by hiteshbheda** [Lomotif/lomotif-ios-issues#752]
Frequency: some times (2 / 10)

Action: 
1. Launch Lomotif app
2. Tap on "START" button
3. Go to camera roll
4. Select more than 100 images (which were captured by iOS device camera)
5. Create lomotif
6. Check whether lomotif created or not and if created then plays smoothly or not

Expected result: Lomotif should be created with selected motifs

Actual result: 
1. Application crash sometimes
2. Black frames display

[Crash logs](https://drive.google.com/drive/u/0/folders/0B6P2U77GNA4UY2s3U1daNzhySHM)

**Comment by testershub** [comment]
@xiuie I can't reproduce this issue for build [1.9.24] on iPhone 6 Plus / 10.0.1 and iPhone 6S / 10.0.1.
Preview is playing smoothly. Also black frames are not displayed.

**Comment by hiteshbheda** [comment]
@xiuie I can reproduce this issue for build 1.9.27 on iPhone 6 / iOS 10.0.2

Application Crashes for most of time for more than 100 images (images should be captured on iOS device camera)

**Comment by xiuie** [comment]
@hiteshbheda I can reproduce this on my iPhone 6/ iOS 10. Might really need to have all camera roll shots.

**Comment by hiteshbheda** [comment]
@xiuie Yes I agree with you. It is reproducible all the times if we create lomotif with more than 100 images captured with iOS device.
